workplace/Living in Open Workspaces.md

```diff
@@ -0,0 +1,78 @@
+9.  Keep an eye on your personal budget. Just because you have an income now doesn’t mean you should “make it rain” all in one place. Invest in your future.
+10. Respect headphones as an indicator of busyness
+11. Find a quiet corner if you want to focus
+12. Intentionally spend some amount of time without headphones on so others can approach you
+13. Send "slack warnings" if you are going to come interrupt someone to reduce surprise
+14. Be sure to link the slideshow, video,
+or other materials you expect them to follow as part of the lesson.
+15. Build on the first information. Have apprentices guess things, do an activity, etc.
+
+## Common Mistakes & Misconceptions
+
+The following actions should be avoided in a workplace:
+
+1.  Don’t “Reply All” to an email chain. Understand the differences—and repercussions—between “Reply” and “Reply All” to avoid humiliation.
+2.  Don’t have a personal conversation at your desk. Find a conference room to discuss after-work issues that you must have with your best friend, sister, significant other, or landlord (or anyone who isn’t related to work, actually).
+3.  Don’t bring your emotions into the office. Your desk neighbor doesn’t want to hear your sob story from the weekend, so leave that at the door.
+4.  Don’t be afraid to ask questions, despite how silly you think they seem. This way, you will avoid erroneously completing an entire project only to realize you did it all wrong.
+5.  Don’t gossip about your coworkers…or your boss. You’re not hurting anyone but yourself when you do. Better yet, don’t be so intolerable that people gossip about you.
+6.  Do not insert emoticons or multiple exclamation points (if any) into work emails. Despite how relaxed your superiors might act, always be professional.
+7.  Do not wear weekend attire to the office. Save the crop tops, flip-flops, and see-through shirts for the weekend. No one will take you seriously if you don’t.
+8.  Don’t apply for a job you don’t want. It will be a waste of time for both parties if you meet the employer for an interview and initially know you will decline the position.
+9.  Don’t talk back to your boss, even if there isn’t much of an age difference between you two. Hopefully, you will have the chance to climb the career ladder someday. You will want people to respect you then, right?
+10. Don’t forget that at work socials, you’re still at work. Be careful not to overdo it if alcohol is being served; everyone will know why you “called in sick” the next day.
+
+## Check for Understanding
+
+Take this [test](http://www.managementcenter.org/tools/quiz/) to check the understanding of your workplace managament skills.
+
+## Supplemental Materials
+
+- [Workplace rules](https://www.northeastern.edu/graduate/blog/workplace-etiquette/)- It consists of the rules to be followed and the mistakes to be avoided at a workplace
```

Comment:
can you make it more clear that you took text from this to create the lesson?  Always cite your sources to avoid plagiarism.

Reply:
In fact, if this is mostly from that article, can you just tell people to read it as the lesson and give them the link?  You can tell them to notice certain things, but copy-pasta here does not seem efficient or helpful.